patterns/hero-product-split.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Title: WooCommerce Hero Product - Split
```

Comment:
I think you can remove the `WooComerce` prefix here. We decided to remove it and all the patterns we are merging don't have it.

Reply:
Updated in 2f38c71.